src/CleanArchitecture.Web/Startup.cs

```diff
@@ -58,14 +51,22 @@ private static IServiceProvider BuildDependencyInjectionProvider(IServiceCollect
             // Populate the container using the service collection
             builder.Populate(services);
 
-            // TODO: Add Registry Classes to eliminate reference to Infrastructure
+            // Reference all CleanArchitecture assemblies for AutoFac.
             Assembly webAssembly = Assembly.GetExecutingAssembly();
             Assembly coreAssembly = Assembly.GetAssembly(typeof(BaseEntity));
```

Comment:
Rather than using BaseEntity as "hook" to reference core assembly we should consider creating a pattern to reference this kind of assemblies for example by an empty class called **InfrastructureReference**, **InfrastructureOrmReference**. A person without knowledge about reflection will have a problem to understand why in this case **BaseEntity** when you will need to add additional Infrastructure's projects.

Reply:
I think a comment would suffice. That's what I usually do is specify a class I know to be in the given assembly and then add a comment with the project/assembly name.